purchases/src/main/kotlin/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -66,6 +69,7 @@ private const val RECONNECT_TIMER_MAX_TIME_MILLISECONDS = 1000L * 60L * 15L // 1
 
 @Suppress("LargeClass", "TooManyFunctions")
 internal class BillingWrapper(
+    private val application: Application,
```

Comment:
I needed a way to access a Context instance in order to retrieve the package info, and this is the simples way I found to do that. Suggestions welcomed tho.